src/index.ts

```diff
@@ -104,6 +104,8 @@ import RuleGenerator from './Component/RuleGenerator/RuleGenerator';
 import RuleTable from './Component/RuleTable/RuleTable';
 import { localize } from './Component/LocaleWrapper/LocaleWrapper';
 import { CompositionContext } from './Component/CompositionContext/CompositionContext';
+import { DefaultValueContext } from './Component/DefaultValueContext/DefaultValueContext';
+import withDefaultValues from './hoc/withDefaultsContext';
```

Comment:
`withDefaultValues` -> `withDefaultsContext`

Reply:
Nice catch, fixed in 8953dea